ops/ccs-ops-misc/load_test/common/db.py

```diff
@@ -22,49 +23,76 @@ def _execute(uri, query):
             with conn.cursor() as cursor:
                 cursor.execute(query)
                 results = cursor.fetchall()
-                print(f'Returned {len(results)} results from the database for the test.')
     finally:
         conn.close()
 
     return results
 
 
-def get_bene_ids(uri):
+def get_bene_ids(uri: str, table_sample_pct: float = 0.25) -> List:
     """
     Return a list of bene IDs from the adjudicated beneficiary table
     """
-    beneQuery = (
+    bene_query = (
         'SELECT "bene_id" '
         'FROM "beneficiaries" '
-        # 'TABLESAMPLE SYSTEM (0.25) '  # when data is cleaned up, use this to access random benes
-        # there are tons of benes with null ref year which return 404 if we use them
+        f'TABLESAMPLE SYSTEM ({table_sample_pct}) '
         'WHERE "rfrnc_yr" IS NOT NULL '
```

Comment:
We shouldnt need the rfrnc_yr not null check here anymore; the application shoudnt error even if theyre null now.